### DISCUSSION TITLE
Feature request: move postings between discussions in categories in the same repo

### DISCUSSION THREAD

**Discussion by andig** [community/community#2913] [Discussions]
Currently one can only transfer to different repos, but to different categories. See https://github.com/andig/evcc/discussions/513#discussioncomment-181786 For an example where user replied to the welcome message instead opening a post in questions category.

**Comment by becca** [comment]
Repo owners, admins and maintainers have the ability to move any Discussion from one category into another. On the right hand side of a discussion click the ⚙️ and change it from there. 
<img width="282" alt="Screen Shot 2020-12-09 at 2 54 03 PM" src="https://user-images.githubusercontent.com/23545426/101686222-6279fb00-3a2e-11eb-8ae8-d12589333266.png">

**Comment by andig** [comment]
This will move the entire discussion, but not a single message. For (re)threading it would be helpful to move single messages. I had expected this in the „...“ symbol.

**Reply by andig** [reply]
I‘ve updated the topic to make this clearer

**Reply by becca** [reply]
Ah thank you for doing that.

**Comment by becca** [comment]
We do not yet have functionality to split/merge conversations within discussions threads but it's a request we have logged and will keep y'all posted when we prioritize this feature.

**Reply by camueller** [reply]
This is really a cruicial feature in order to keep discussions organized. Not everybody cares about the topic being used to post. This leaves the maintainer to move single post to existing/new discussions in the same repository.

**Reply by ricardoevvargas** [reply]
I agree.

**Comment by ricardoevvargas** [comment]
Is this feature expected to be implemented?

**Comment by ARGAMX** [comment]
Any update on this? Is this feature expected to be implemented?
